backend/online-bookmark/api/common/handle-simple-query.js

```diff
@@ -0,0 +1,12 @@
+module.exports = handleQuery;
+
+function handleQuery(db, res, err, docs) {
```

Comment:
Just a question: shouldn't we follow the `function (err, arg1, arg2...) { ... }` schema?